k2/csrc/fsa_util.cc

```diff
@@ -201,4 +213,83 @@ std::string FsaToString(const Fsa &fsa) {
   return os.str();
 }
 
+int32_t Rand(int32_t low /* = std::numeric_limits<int32_t>::min() */,
```

Comment:
Maybe we should add some comments on `Rand` to say that it is not suggested to call it many times in one function, as it will create `gen` and `dis` again and again. Anyway, it's OK we call it in `GenerateRandFsa` as it's just a testing function

Reply:
A class `RandInt` for generating random numbers is implemented.